src/Core/Scheduler/Scheduler.js

```diff
@@ -14,7 +16,16 @@ import PointCloudProvider from '../../Provider/PointCloudProvider';
 import WFSProvider from '../../Provider/WFSProvider';
 import RasterProvider from '../../Provider/RasterProvider';
 import StaticProvider from '../../Provider/StaticProvider';
-import CancelledCommandException from './CancelledCommandException';
+
+import GpxParser from '../../Parser/GpxParser';
+import XbilParser from '../../Parser/XbilParser';
+import GeoJsonParser from '../../Parser/GeoJsonParser';
+import PntsParser from '../../Parser/PntsParser';
+import B3dmParser from '../../Parser/B3dmParser';
+import PotreeBinParser from '../../Parser/PotreeBinParser';
+import PotreeCinParser from '../../Parser/PotreeCinParser';
+// import GLTFLoader from '../../Parser/GLTFLoader';
```

Comment:
Do you plan to support it before merging ? If not, please remove those comments.

Reply:
Ok, I will remove them

Reply:
done